source/js/buyers-guide/analytics-events.js

```diff
@@ -77,43 +41,20 @@ function getQuerySelectorEvents(pageTitle, productName) {
       action: `opinion submitted`,
       label: `opinion on ${productName}`
     },
-    "#product-social-button-fb": {
-      category: `product`,
-      action: `share tap`,
-      label: `share ${productName} to Facebook`,
-      transport: `beacon`
-    },
-    "#product-social-button-twitter": {
-      category: `product`,
-      action: `share tap`,
-      label: `share ${productName} to Twitter`,
-      transport: `beacon`
-    },
-    "#product-social-button-email": {
-      category: `product`,
-      action: `share tap`,
-      label: `share ${productName} to Email`,
-      transport: `beacon`
-    },
-    "#privacy-policy-link": {
+    "a.privacy-policy-link": {
       category: `product`,
       action: `privacy policy link tap`,
       label: `policy link for ${productName}`,
       transport: `beacon`
     },
-    "#reading-level-link": {
-      category: `product`,
-      action: `carnegie mellon reading level links`,
-      label: `reading level link for ${productName}`,
-      transport: `beacon`
-    },
 
     // product updates
     ".product-update-link": {
       category: `product`,
       action: `update article link tap`,
       label: `update article link for ${productName}`,
-      transport: `beacon`
+      transport: `beacon`,
+      optional: true
```

Comment:
`optional` doesn't seem to be a [GA Event option](https://developers.google.com/analytics/devguides/collection/analyticsjs/events). Can we add inline comment to explain what it is for?